pkg/app/app.go

```diff
@@ -14,6 +15,27 @@ type App struct {
 
 // New bootstrap a new application.
 func New(opts command.Options) (*App, error) {
+	var sshConfig *sshdb.SSHConfig
+
+	if opts.SSHHost != "" {
+		sshConfig = sshdb.New(
+			sshdb.WithDBDriver(opts.Driver),
+			sshdb.WithSShHost(opts.SSHHost),
+			sshdb.WithSShPort(opts.SSHPort),
+			sshdb.WithSSHUser(opts.SSHUser),
+			sshdb.WithPass(opts.SSHPass),
+			sshdb.WithSSHKeyFile(opts.SSHKeyFile),
+			sshdb.WithSSHKeyPass(opts.SSHKeyPassphrase),
+			sshdb.WithDBDURL(opts.URL),
+		)
+
+		if err := sshConfig.SSHTunnel(); err != nil {
+			return nil, err
+		}
+
+		defer sshConfig.Close()
```

Comment:
will this result in the ssh tunnel closing before the application closes?

Reply:
You're right. I messed it up.

Already fixed on #249

Please, check it out!